pootle/apps/contact/forms.py

```diff
@@ -56,6 +57,17 @@ def from_email(self):
     def recipient_list(self):
         return [settings.POOTLE_CONTACT_EMAIL]
 
+    def save(self, fail_silently=False):
+        """
+        Build and send the email message.
```

Comment:
Nit: mind the [docstring conventions](http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#docstrings). Same elsewhere.